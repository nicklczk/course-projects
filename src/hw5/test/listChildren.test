# Creates graph
CreateGraph graph0

# Adds a node to a graph
AddNode graph0 node0

# Adds another node to a graph
AddNode graph0 node1

# Add edge from parent node to child node in graph
AddEdge graph0 node0 node1 zeroToOne

# Lists children of parent node in graph
ListChildren graph0 node0