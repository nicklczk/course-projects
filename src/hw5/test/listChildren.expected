# Creates graph
created graph graph0

# Adds a node to a graph
added node node0 to graph0 

# Adds another node to a graph
added node node1 to graph0

# Add edge from parent node to child node in graph
added edge zeroToOne from node0 to node1 in graph0

# Lists children of parent node in graph
the children of node0 in graph0 are: node1(zeroToOne)